src/components/flippable/Flippable.tsx

```diff
@@ -0,0 +1,115 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+export interface IFlippableOwnProps {
+    id?: string;
+    front?: React.ReactNode;
+    back?: React.ReactNode;

```

Comment:
Shouldn't front and back be mandatory? Else you'll be displaying `undefined` below.

Reply:
Well, if undefined, the divs for the front and/or the back sides will simply be rendered empty, which won't cause any error. However, it doesn't make that much sense to render an empty flippable. I don't really know if there is a use case for this.

Reply:
It would be safer to just add default props of empty nodes.